viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -2,7 +2,8 @@
  * Copyright 2022 Cognite AS
  */
 
-import { CameraChangeDelegate, CameraState } from './types';
+import { CameraState } from './types';
+import { CameraChangeDelegate } from '@reveal/utilities';
```

Comment:
It seems very strange to me that the camera change delegate is a part of utilities.

Reply:
Move to CameraManager package